test/unit/sql/enum.test.js

```diff
@@ -24,6 +25,22 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
         }
       });
 
+      describe('pgEnumName', function() {
+        it('does not add schema when options: { schema: false }', function() {
+          expect(sql.pgEnumName(PublicUser.getTableName(), 'mood', { schema: false }))
+            .to.equal('"enum_users_mood"');
+          expect(sql.pgEnumName(FooUser.getTableName(), 'theirMood', { schema: false }))
+            .to.equal('"enum_users_theirMood"');
```

Comment:
When would a user specifically use `schema: false`? Would it not normally just be unset?